dotcom-rendering/src/components/Carousel.importable.tsx

```diff
@@ -413,17 +413,23 @@ const Title = ({
 	titleColour,
 	titleHighlightColour,
 	isCuratedContent,
+	url,
 }: {
 	title: string;
 	titleColour: string;
 	titleHighlightColour: string;
 	isCuratedContent?: boolean;
+	url?: string;
 }) =>
-	title === 'Videos' ? (
+	url ? (
 		<a
 			css={[linkStyles]}
-			href="https://www.theguardian.com/video"
-			data-link-name="video-container-title Videos"
+			href={url}
+			data-link-name={
+				title === 'Videos'
+					? 'video-container-title Videos'
+					: 'section heading'
```

Comment:
This is used for all other [container title links ](https://github.com/guardian/dotcom-rendering/blob/ec5672f2d2ca04198f8ee89a0d779caa016d9fb6/dotcom-rendering/src/components/ContainerTitle.tsx#L114)so I've used it here as well to help standardise.